_posts/2021-05-01-sr2021-knockouts.md

```diff
@@ -0,0 +1,190 @@
+| 3<sup>rd</sup> Place  | MCK: [The Malay College Kuala Kangsar](https://mckk.edu.my)
+| <span class="has-explanation" title="The team that displays the most extraordinary ingenuity or elegant simplicity in the design of their robot">Committee Award</span> | HRS2: [Hills Road Sixth Form College](https://www.hillsroad.ac.uk) Team 2
+| <span class="has-explanation" title="The rookie team who achieve the highest place in the league">Rookie Award</span> | WER: [We Robot](https://www.werobot.fr)
+| <span class="has-explanation" title="The team that is judged to have the best online presence">Online Presence</span> | YSH: [Yayasan Sultan Haji Hassanal Bolkiah](https://www.secondary-yayasan.edu.bn/en/)
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2021/trophy-render.png"
+           alt="A podium, with rendered trophies, featuring SPA in Gold, HRS2 in Silver, and MCK in Bronze"
+           caption="The podium positions" %}
+
+With a consistently strong performance, scoring well in the first league and
+really stepping up their game in the final league session French team We Robot
+ended the league stage with the highest number of game points of any team. While
+this didn’t place them at the top of the league leaderboard, they were the
+highest placed rookie team and thus earned the Rookie Award.
+
+The Committee Award is given for extraordinary ingenuity or simple elegance in
+the design of their solution. Team HRS2 from Hills Road Sixth Form College
+earned this prize in recognition of the high quality of the code that they wrote
+for their robot. In particular it was clean and easy to read, neatly organised
+in separate files, with good comments. Their code made good use of a state
+machine as part of its main control loop and PID control for their movement.
+
+We also encourage teams to share their progress towards their robots throughout
+the year. Team YSH from Yayasan Sultan Haji Hassanal Bolkiah earned the Online
+Presence Award for their [Instagram posting][ysh-instagram] throughout the year
+covering their approach to the competition, strategy and reviews of their
+performance in the leagues.
+
+For full details on all the awards, please see the [rulebook][rulebook].
+
+You can see a breakdown of scores for each match, as well as the overall league
+ranking on the [competition website](/comp/).
+
+Rewatch the streams
+-------------------
+
+If you’d like to relive the highs and lows of the competition livestreams, the
+videos remain available on YouTube, as well as a cut-down video of the final:
+
+- [League Session 1](https://www.youtube.com/watch?v=cAvk-nfTUis)
+- [League Session 2](https://www.youtube.com/watch?v=RwW5Oz30gbE)
+- [League Session 3](https://www.youtube.com/watch?v=MpuhtW4mCKM)
+- [League Session 4](https://www.youtube.com/watch?v=E9gF-GZbf5M)
+- [Knockouts and Final](https://www.youtube.com/watch?v=gxJM6ernMqo)
```

Comment:
Given this is a post about the knockouts and finals, should we embed this as an actual player?

Reply:
We already have one embed here and this article is pretty long (as is the livestream). I think we're fine to just leave this as a link.